harvest/broker/_base.py

```diff
@@ -123,12 +154,31 @@ def wrapper(*args, **kwargs):
             raise Exception(f"{func} failed")
         return wrapper
 
-    def fetch_price_history( self,
-        last: dt.datetime, 
-        today: dt.datetime, 
-        interval: str='5MIN',
-        symbol: str = None):
-        """Returns historical price data for the specified asset and period.
+    # def fetch_price(self,
+    #     start: dt.datetime, 
+    #     end: dt.datetime, 
+    #     interval: str='5MIN',
+    #     symbol: str = None):
+    #     """Returns historical price data for the specified asset and period
+    #     from storage and may call fetch_price_history to populate the storage.
+
+    #     :last: The starting date of the period, inclusive.
+    #     :today: The ending date of the period, inclusive.
+    #     :interval: The interval of requested historical data.
+    #     :symbol: The stock/crypto to get data for.
+
+    #     :returns: A pandas dataframe, same format as _handler()
+    #     """
+    #     raise NotImplementedError("This endpoint is not supported in this broker")
```

Comment:
This was temporarily removed since it was not used

Reply:
Looking over the code again I see why this was added. Though I still think it should be disabled temporarily for four reasons:
- Now that `storage` has moved to `Trader`, the implementation becomes a bit messier.
- It should be the `Trader`'s job to cache data
- Its functionality overlaps with `BaseStorage.load()` and is a little confusing. 
- Chances are, it will have to call `fetch_price_history` almost every time anyways since price data is updated every minute.

Reply:
I fully agree. I removed this from my own refactor.

Reply:
:)